resources/active_directory.rb

```diff
@@ -28,6 +28,7 @@
 
 resource_name :sensu_active_directory
 provides :sensu_active_directory
+unified_mode true
```

Comment:
what is the min version of chef is required to use this?

Reply:
Unified Mode is unavailable prior to 14.14 [according to the docs](https://docs.chef.io/unified_mode/).  I've been adding these to appease cookstyle in my dev environment but isn't strictly necessary for this change.